<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainKitapKayit</class>
 <widget class="QMainWindow" name="MainKitapKayit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kitap kayıt bilgileri</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_kayit">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kütüphane kayıt numarası</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_kitap">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kitabın adı</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_yazar">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Yazarı</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_sayfa">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sayfa sayısı</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txt_kayit">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txt_kitap">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txt_yazar">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txt_sayfa">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_tamam">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kaydet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_iptal">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>İptal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>457</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
